addons/full-signature/titles.css

```diff
@@ -0,0 +1,4 @@
+.preview .project-title.no-edit {
+  overflow-x: auto;
+  text-overflow: clip;
```

Comment:
I'd add `scrollbar-width: thin` here too - the default scrollbar uses a significant amount of vertical space on some browsers.